src/mono/wasm/runtime/driver.c

```diff
@@ -24,6 +24,8 @@ void core_initialize_internals ();
 #endif
 
 // Blazor specific custom routines - see dotnet_support.js for backing code
+extern void* mono_wasm_invoke_js_blazor (MonoString **exceptionMessage, void *callInfo, void* arg0, void* arg1, void* arg2);
```

Comment:
The reason for the `arg0`/`arg1`/`arg2` params is so we can retain the earlier semantics about how reference and value types are represented in the unmarshalled interop cases.